lib/iris/cube.py

```diff
@@ -1627,9 +1627,17 @@ def dtype(self):
     @property
     def fill_value(self):
         """
-        The fill value for the masked data array of this
-        :class:`~iris.cube.Cube`. If ``None``, the default NumPy fill value for
-        the dtype of the masked data array will be used.
+        A value suitable for filling in missing (i.e. masked) datapoints of
+        this :class:`~iris.cube.Cube` s data.
+
+        This can also be ``None``, meaning that no suitable fill value is
```

Comment:
Can you remove 'also' please, and add this to the former paragraph? I think this line is part of the same concept as the above sentence.